### PR TITLE
Bosch intergration

### PR DESCRIPTION
**Pull request by VincentKgadima**
This includes the uplink decoder, add device user interface and the node-red config downloaded for Multitech gateway

### CODE REVIEW

File: client/components/addDevice.js [hunk 1]
```diff
@@ -36,7 +36,99 @@ export default class AddDevice extends Component {
         wifipass: "",
         code: [],
         pairedDevices: [],
-        bleDevices: []
+        bleDevices: [],
+        value:
```
Comment (fluentart) [inline client/components/addDevice.js]: This looks like Bosch specific code, please move this into the Bosch plugin. We have to keep addDevice seperate from device specific code else things will get crazy in the future.

Comment (VincentKgadima) [reply client/components/addDevice.js]: Moved in to config.json

**Comment by fluentart** [comment]
Hi Vince theres a conflict before I can merge this.

**Comment by VincentKgadima** [comment]
@fluentart Merge conflicts resolved

**Pull request merged**